ext/dogstatsd_client.c

```diff
@@ -46,60 +48,40 @@ void ddtrace_dogstatsd_client_rinit(void) {
                     LOG(WARN,
                         "Dogstatsd client encountered an invalid udp:// DD_DOGSTATSD_URL: %s, missing a colon followed by a port",
                         url);
+                    efree(url);
```

Comment:
free(), not efree - it's not allocated via emalloc.

Reply:
I used ZVal to store the url instead